### DISCUSSION TITLE
Video tutorial in French "Prise en main d'OpenModelica" (getting started with OpenModelica)

### DISCUSSION THREAD

**Discussion by pierre-haessig** [OpenModelica/OpenModelica#9786] [General]
Dear OpenModelica community,

In case it's may be useful to some other people, I happen to have recorded three 10 to 15-minute videos *in French* on getting started with OpenModelica:

- YouTube playlist: https://www.youtube.com/playlist?list=PL85o3JxSBDh3hujVv0i-WDsVvdm_s8wVf
- Dedicated page for these videos on my course website : http://éole.net/courses/modelica/81-OM-start.html

Now, this is not, as I discovered afterwards, the first YouTube tutorial about OpenModelica in French (see e.g. the work of [InSyTe.website ](https://www.youtube.com/watch?v=WLLaB_Ter_s&list=PL_ai74iFIdv6AxGtCbk6yyBbvutv5HC9U&index=9) and [Alain Charbonnel](https://www.youtube.com/watch?v=r5esuiq1Uzc)), but I would say mine are not meant as a resource to learn Modelica, but rather focusing on providing the basic elements needed to practice while learning from another source (I'm thinking here of Michael Tiller’s book [mbe.modelica.university/](https://mbe.modelica.university/) which is tool-agnostic).

**Comment by casella** [comment]
Good! We should link it to openmodelica.org
